js/app/test/slider_release.spec.ts

```diff
@@ -47,6 +43,6 @@ test("slider release", async ({ page }) => {
 
 	const val = await slider.inputValue();
 	expect(parseInt(val)).toBeCloseTo(70);
-	expect(value).toHaveValue("70");
+	expect(parseInt(await value.inputValue())).toBeCloseTo(70);
```

Comment:
Now we are testing the actual UI there is less accuracy here but this works nicely.